### DISCUSSION TITLE
생각하고 있는 디자인&기획 개선 범위 질문

### DISCUSSION THREAD

**Discussion by rbgus2002** [rbgus2002/cactusdy#206] [Q&A] [answered]
접때 말씀해주셨지만, 특히 홈화면이 마음에 가장 안든다고 하셨는데 추가적으로 어떤 부분에서 개선이 필요하다고 느끼시나요??
저는 개인적으로 BE 입장에서 형태는 비슷하게 가져가되, 전면적으로 디자인이 수정됐으면 하는 마음이 있습니다. 그치만 FE 쪽에서 공수가 큰 작업이니... 의견 존중하겠습니다ㅋㅋ

저는 아래 항목들이 시급하다고 생각합니다.
1. 홈화면 전반
2. 앱에서 색상과 관련된 부분들이 애매모호한 거 같아요. 앱 메인 컬러가 초록색 && 스터디 색상 지정 가능 && 태그 색상 다양함 등 때문에 애매모호하다고 느끼는데 생각하다 보니까 스터디 색상을 지정할 수 있게 해야하나도 싶습니다. 사실 스터디한테 색상 선택이라는 기능을 부여한 이유는 스터디 마다 구분이 쉽도록 하기 위함이었는데, 프로필 사진 + 배경사진을 설정할 수 있게 하면 그걸로 충분히 구분히 가능하지 않나 싶습니다. 그래서 스터디 색상 선택 기능을 제거하고 이 부분 다시 생각해서 새롭게 기획&디자인 원해요
3. 회차 상세페이지의 `기록`이 현재는 회차 참여인원이 공통으로 보여지는 영역인데, 개인만 따로 기록을 사용할 수 있는 영역도 있으면 좋겠습니다.

이야기 대강 나눠보고  디자이너&기획 맡아줄 수 있는 분 구해보고 싶어요 ㅎㅎ

**Comment by Arkady1356** [comment]
1. 동의합니다!!
2. 스터디 색상 제거는 어려울것 같습니다..!
  1) 처음부터 안했으면 인정인데 줬다 뺏으면 불만족도 상승이 큽니다..! (소비자행동론::전망이론: 긍정적 변화보다 부정적 변화에 민감하게 반응)
  2) 이건 그냥 의견인데 초록색만 있으면 그냥 네이버 앱 같은 디자인 될텐데 별로 안이쁘다고 생각합니다 ㅎㅎ.. 네이버 앱 > 왼쪽 상단 = 눌렀을때 보이는 화면 보고 이쁜지 생각해보세요..!
  3) 대신 컬러칩 개념은 빼도 나쁘지 않을 것 같아요 >> 출석 체크 잘하는지도 궁금하다
3. 기록은 흠 ...

개인 기록의 필요성 인정
하지만 최초의 취지는 그날 있었던 약속, 새로운 규칙, 사건 등을 잊지 않기 위해 기록하는 용도라서.. 공통인것이 맞다고 생각하고
개인으로 하면 쓰는 사람만 쓸 것 같은 느낌??
-> 작성하는 내용이 공개적으로 공유될 필요성이 어느정도 있다고 생각
-> 공통 기록에 댓글처럼 개인 기록을 추가로 comment할 수 있도록 하고 모두에게 보여지게 한다면?

**Reply by Arkady1356** [reply]
추가로 전에 말했던 스터디 색상 기본값으로 쌈무색인거 아직 그대로인 것 같아..!
전에 말한것 처럼 열심히 꾸몄는데 다른사람한테 전달되지 않으면 안된다고 생각하고,
대부분 주체자가 하자 해서 시작하는 것일 텐데 귀찮아서 안바꿀텐데 아무튼 방장이 선택한 컬러로 기본값 가져갔으면 좋겠다

**Reply by rbgus2002** [reply]
>추가로 전에 말했던 스터디 색상 기본값으로 쌈무색인거 아직 그대로인 것 같아..!
전에 말한것 처럼 열심히 꾸몄는데 다른사람한테 전달되지 않으면 안된다고 생각하고,
대부분 주체자가 하자 해서 시작하는 것일 텐데 귀찮아서 안바꿀텐데 아무튼 방장이 선택한 컬러로 기본값 가져갔으면 좋겠다

오잉 그거 해놨는데... 테스트까지 했던 부분인데 이상하네요
테스트 해보셨을 때 안돼서 말씀하신건가요??

**Reply by rbgus2002** [reply]
> 처음부터 안했으면 인정인데 줬다 뺏으면 불만족도 상승이 큽니다..! (소비자행동론::전망이론: 긍정적 변화보다 부정적 변화에 민감하게 반응)

홈화면도 전면적으로 개선하는 이 시점에서, 기능 추가/삭제에 현재 사용자들까지 크게 신경써야하나 싶습니다. 아직 사용자가 적기 때문에 드리는 말씀이었고, 아직은 ui/ux 적으로 미흡한 점이 보여 저는 현재 뜨끔이 베타 버전에 있다고 생각해요

**Reply by rbgus2002** [reply]
> 개인 기록의 필요성 인정
하지만 최초의 취지는 그날 있었던 약속, 새로운 규칙, 사건 등을 잊지 않기 위해 기록하는 용도라서.. 공통인것이 맞다고 생각하고
개인으로 하면 쓰는 사람만 쓸 것 같은 느낌??
-> 작성하는 내용이 공개적으로 공유될 필요성이 어느정도 있다고 생각
-> 공통 기록에 댓글처럼 개인 기록을 추가로 comment할 수 있도록 하고 모두에게 보여지게 한다면?

네 저도 최초의 취지는 유지되어야 한다고 생각해서 공통인 영역은 그대로 두고 개인이 따로 기록할 수 있는 부분이 마련되었으면 좋겠습니다 !  이 기능은 사소하다고 생각 해서 이 부분 디자인과 기획은 (디자이너와 함께) 나중에 고민해보시죠

**Comment by rbgus2002** [comment, marked as answer]
말씀하시는 거 들어보고 좀 찾아보니까 스터디 색상 기능이 마음에 안들었던 이유가 미흡한 디자인 때문이라고 느껴집니다! 카카오뱅크도 보니까 통장마다 색상을 선택할 수 있던데 이 부분 레퍼런스로 삼아서 참고 좀 해봐도 좋을 거 같네요. 그러면 스터디 색상은 그대로 둘게요
아래처럼 정리하겠습니다~

1. 홈 화면 전반적으로 개선
2. 스터디 색상 현행 유지 및 디자인 개선
3. 개인 기록은 애매함 => 추후 다시 생각해보기
